src/pages/index.js

```diff
@@ -168,22 +307,15 @@ cardModalbtn.addEventListener("click", () => {
   openModal(cardModal);
```

Comment:
```js

cardModalbtn.addEventListener("click", () => {
  editModalNameInput.value = profileName.textContent;
  editModalDescriptionInput.value = profileDescription.textContent;
  openModal(cardModal);
});

```


Please, delete the 2 lines of code (305 and 306) because they are for the **profile** opening